centos-7-minimal.ks

```diff
@@ -65,13 +75,52 @@ shim
 LANG="en_US"
 echo "%_install_lang $LANG" > /etc/rpm/macros.image-language-conf
 
+# sudo permission
+echo "%docker ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers.d/docker
+sed -i "s/^.*requiretty/#Defaults requiretty/" /etc/sudoers
+
+cat > /etc/rc.d/init.d/copy-ssh << EOF
+#!/bin/sh
+
+# TODO Generate Locale. This needs further investigation. I'd prefer to disable
+# 'AcceptEnv LANG LC_*'' in '/etc/ssh/sshd_config', but it did not work initially
+localedef -v -c -i en_US -f UTF-8 en_US.UTF-8
+
+# TODO Why can I not generate the docker user outside of this init script as part
+# of the kickstart file?
```

Comment:
@hferentschik we should create a separate issue to track this issue. Not sure why you can not do that . Need more investigation.